tests/test_data/write_specified_cartesian_speaker.accepted.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ebuCoreMain xmlns:dc="http://purl.org/dc/elements/1.1/" xmlns="urn:ebu:metadata-schema:ebuCore_2014" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" schema="EBU_CORE_20140201.xsd" xml:lang="en">
+	<coreMetadata>
+		<format>
+			<audioFormatExtended>
+				<audioChannelFormat audioChannelFormatID="AC_00011001" audioChannelFormatName="Test" typeLabel="0001" typeDefinition="DirectSpeakers">
+					<audioBlockFormat audioBlockFormatID="AB_00011001_00000001" rtime="00:00:01.000000000" duration="00:00:01.000000000">
+						<speakerLabel>testLabel</speakerLabel>
+						<position coordinate="X" screenEdgeLock="left">0.000000</position>
+						<position coordinate="X" bound="min">-0.100000</position>
+						<position coordinate="X" bound="max">0.100000</position>
+						<position coordinate="Y">0.000000</position>
+						<position coordinate="Y" bound="min">0.100000</position>
+						<position coordinate="Y" bound="max">-0.100000</position>
+						<position coordinate="Z">0.500000</position>
+						<position coordinate="Z" bound="min">0.400000</position>
+						<position coordinate="Z" bound="max">0.600000</position>
+					</audioBlockFormat>
+				</audioChannelFormat>
+			</audioFormatExtended>
+		</format>
+	</coreMetadata>
+</ebuCoreMain>
+
```

Comment:
Helps if I highlight the whole thing.

Reply:
yeah that looks good to me (or at least it matches the test cases in the EAR)